### DISCUSSION TITLE
Frontend - Mobile Consistency

### DISCUSSION THREAD

**Discussion by kubraaksux** [bounswe/bounswe2023group1#359] [General]
Web Figma: https://www.figma.com/file/LmUQJGetWO7oD8qCHAwtbl/ResQ?type=design&node-id=542-696&mode=design&t=2iCghi1uzQq1KVAN-0
Mobile Figma: https://www.figma.com/file/LmUQJGetWO7oD8qCHAwtbl/ResQ?type=design&node-id=597-4381&mode=design&t=2iCghi1uzQq1KVAN-0

There are confusions and inconsistencies with mockups.

- Did we planned users to have usernames? Because our "sign in" has username part.
- What will be our landing page? Sign in? Sign up? Map without registration? Or choosing the role type? It is not clear and it should be as soon as possible.
- Frontend and mobile team didnt conclude the design choices with the functional ones like the one above.
Those are the issues i have come so far. They needed to be solved and i believe there is more mockup problems that needed to be found and solved then.

**Comment by HarunErgen** [comment]
- "username" should be removed. We won't need or keep such information. While registration, automatic ID assignment could be made on the backend to distinguish users.
- Landing page should be "Sign in" page.

**Comment by kubraaksux** [comment]
I implemented landing page as signup after discussed with some team members especially backend. Then we discussed with frontend team and because of app's urgent help tendency, last implementation was choosing landing page as map. This has to be concluded especially btw front-mobile. 
Also after the customer meeting (milestone presentation) from the feedback of customers i conclude that quick help, quick info of resources can be more preferrable. 
Agreed on username. We also removed it. Also mockups are updated according to that

**Comment by kubraaksux** [comment]
Again, there is still inconsistencies with mobile which should be considered and discussed

**Comment by kubraaksux** [comment]
Not standardized theme also inconsistency btw mobile and front. And i suggest to use ResQ logo with its original color red.

**Comment by kubraaksux** [comment]
- landing page
- color, theme
- at signup web: Already have an account? Sign in -- mobile: Already have an account?  Log in
- at sign in web: Don't have an account? Sign Up--  mobile: Not Registered Yet? Create an account
- at web and sign in "Forgot password?" is under the sign in at mobile it is above the sign in.
- mobile: I accept the terms and privacy policy. web: By signing up, you agree to our Terms , Privacy Policy and Cookies Policy .
- request views and page icons are different
- requests have priority and type at web and mobile. at web additionally they have status.
- user profile significantly different at mobile and web. also web calls it account. 
- my requests, notifications and settings not decided at web yet.
- role shouldnt be selected at log in, mobile should update this. this was a team decision.
-

**Reply by HarunErgen** [reply]
- Landing page might be the map page with guest view with login and signup options rather than login or registration pages.
- We can open a poll for the all team. I think mobile colors are better.
- Will be changed to "Sign in"
- Will be changed to "Don't have an account? Sign Up"
- No big deal in my opinion. If it is considered important, it can be located below.
- "I accept the terms and privacy policy" this statement is better in my opinion.
- Fields of the requests will be finalized. "Status" wil be added.
- Account on mobile will be corrected according to web.
- We removed it while implementing. We will also remove it from mockups.

**Comment by kubraaksux** [comment]
Mobile team decided not to work parallel with the frontend team unfortunately.  I tried my best to find a common way but couldnt make it. 
With all due disrespect to my work i do not want to pursue this and im closing this issue too.
[you can check](https://github.com/bounswe/bounswe2023group1/issues/269)

**Comment by HarunErgen** [comment]
@kubraaksux We can make changes to keep in line on functionality. I have expressed my requests through the WhatsApp platform to the web team for color palettes and similarity in design, but they were not taken into consideration. I guess it was overlooked because I didn't share it on Github. For now we can focus on functionality.

**Comment by alperenDagi** [comment]
As the mobile team, we couldn't begin our work until the designs were finalized. We completed the design phase early, and it has been available to you for a few weeks. If you want us to follow the design you've created, I believe it's important for you to start your work early and communicate this to us. Expecting us to develop applications for demo right before the milestone while also making design changes isn't practical. Let's avoid unnecessary drama and time-wasting. Effective time management, by starting early, can help ensure things progress as desired. Otherwise, we can't be held responsible for last-minute requests.

**Comment by kubraaksux** [comment]
I think communication problem wasn't about us, as i documented this many times and discussed at labs. 
Also, i really don't appreciate personal approaches towards me for a few comments now. 
I was mentioning the problems since mobile team shared their designs. The reason behind the fact web was not early as mobile is we were deciding the mechanism and we had discussed those with backend at the labs and when those happening you were at the early stages of mockups. We have communicated with backend so that we could have finalized our minds and at this duration also tried to communicate and inform mobile. Also we first needed to see our first page at web to examine our components of our libraries, so that updating the designs in a much better way. 
Im trying to solve this problem for three weeks. [check this issue](https://github.com/bounswe/bounswe2023group1/issues/241) and your designs are 2 weeks old. Again, our frontend development started 2-3 weeks before and it was open to review. My purpose was not making you follow us, finding a common way btw back-front-mobile. 
If you had checked my designs, you could also see that i tried to make everything components one by one for the others till little buttons or logos. 
If you will have harsh comments and if you'll continue to close issues without value other's opinions, you need to be sure you have checked other's works, made discussions and following issues. Also lab times can be used for discussions, not github..

**Comment by ilgazer** [comment]
@kubraaksux @HarunErgen @alperenDagi from what I have gathered with all the different places this discussion has taken place, every one of us agrees that the two sets of mockups should be functionally equivalent to each other. I think that the best course of action from now on is to list in this thread where the two sets of mockups diverge from each other. That way, we can rectify those issues, and also address the vague requirements that lead to this divergence.
